org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/quickfix/XbaseQuickfixProvider.java

```diff
@@ -236,6 +238,20 @@ public void apply(EObject element, IModificationContext context) throws Exceptio
 					});
 		}
 	}
+	
+	@Fix(IssueCodes.UNUSED_LOCAL_VARIABLE)
+	public void removeUnusedLocalVariable(final Issue issue, IssueResolutionAcceptor acceptor) {
+		// use the same label, description and image
+		// to be able to use the quickfixes (issue resolution) in batch mode
+		String label = "Remove.";
+		String description = "Remove the unused local variable.";
+		String image = "fix_indent.gif";
```

Comment:
Maybe use fix_local_var.gif instead.

Reply:
We will use the remove icon from jdt instead.

Reply:
Thanks for the feedback, it is a good idea to use the same symbols as jdt uses. I updated the PR as requested.